### PR TITLE
Translating code and improving Spanish terminology

### PR DESCRIPTION
**Pull request by raydelto**
The term previously used to refer to arrays was the Spanish word for matrix, which was inaccurate since not all arrays are matrices.

I've also translate the code and its comments to Spansih

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

File: guide/spanish/cplusplus/arrays/index.md [hunk 1]
```diff
@@ -2,41 +2,38 @@
...
 ```C++
-//Initialization with entries: 
- int numbers [5] = {1, 2, 3, 4, 5}; 
+ //Inicializando el arreglo con valores previos
+ int numeros[] = {1, 2, 3, 4, 5}; 
  
- //Initialization with no values: 
- int numbers [5] = {}; 
+ //Inicializando el arreglo sin datos
+ int numeros[5];
```
Comment (lynxlynxlynx) [inline guide/spanish/cplusplus/arrays/index.md]: While the effect is the came in c++ (invalid in c), I think it'd be better to leave the explicit initialization in.

File: guide/spanish/cplusplus/arrays/index.md [hunk 2]
```diff
@@ -2,41 +2,38 @@
 title: C++ Arrays
 localeTitle: Arrays C ++
```
Comment (lynxlynxlynx) [inline guide/spanish/cplusplus/arrays/index.md]: missing translation

**Comment by RandellDawson** [comment]
@lynxlynxlynx Do you want to fix the issues or close this as stale?

**Comment by lynxlynxlynx** [comment]
It's fine as-is.

**Review by RandellDawson — APPROVED** [review]

**Comment by RandellDawson** [comment]
Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 🎉

**Pull request merged**